onnx/checker.py

```diff
@@ -171,4 +171,14 @@ def check_model(
         C.check_model(protobuf_string, full_check, skip_opset_compatibility_check)
 
 
+def resolve_external_data_location(
+    base_dir: str, location: str, tensor_name: str
+) -> str:
+    result = C.resolve_external_data_location(base_dir, location, tensor_name)  # type: ignore[attr-defined]
+    if result is None:
```

Comment:
Should this be private? Do we want to expose this to users?

Reply:
this is for ORT to use from its c++ code.

Reply:
removed